src/esp/gfx/DrawableGroup.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#include "DrawableGroup.h"
+#include "Drawable.h"
+
+namespace esp {
+namespace gfx {
+
+class Drawable;
+
+DrawableGroup::~DrawableGroup() {
+  // has to inform any currently existing drawables, do not query anything
+  // regarding this group
+  for (int iDrawable = 0; iDrawable < this->size(); ++iDrawable) {
+    static_cast<Drawable&>((*this)[iDrawable]).groupExists_ = false;
```

Comment:
I do not like this solution actually.
Look at this static_cast here, it means the DrawableGroup can only work with our Drawable class, not any magnum original drawables. 
(I think it is OK, just like our scene graph will only work with our own scene node. Do you agree?)